data/Language/C.yaml

```diff
@@ -0,0 +1,62 @@
+identifier: C
```

Comment:
You can remove this commit: 6b050c171f17235794b2f8e59b8b13dba1d7514b from this branch. It is not related to your PR. Use `rebase` to drop the commit.